src/enzyme_ad/jax/Passes/EnzymeHLOOpt.cpp

```diff
@@ -12765,6 +12765,48 @@ struct AbsPositiveSimplify : public OpRewritePattern<stablehlo::AbsOp> {
   }
 };
 
+struct AbsSimplify final : public OpRewritePattern<stablehlo::AbsOp> {
+  using OpRewritePattern<stablehlo::AbsOp>::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(stablehlo::AbsOp op,
+                                PatternRewriter &rewriter) const override {
+    // Match when the operand is a constant.
```

Comment:
can you re-use: https://github.com/EnzymeAD/Enzyme-JAX/blob/0564086aa34d92d9fa94166e482b1dfa3348474e/src/enzyme_ad/jax/Passes/EnzymeHLOOpt.cpp#L3455